From 070a389ae536a75b9184784f625949c215c533b6 Mon Sep 17 00:00:00 2001
From: John Crispin <blogic@openwrt.org>
Date: Thu, 23 May 2013 18:50:56 +0200
Subject: [PATCH 09/33] MIPS: ralink: workaround DTB memory issue

If the DTB is too big a bug happens on boot when init ram is freed.
This is a temporary fix until the real cause is found.

Signed-off-by: John Crispin <blogic@openwrt.org>
---
 arch/mips/ralink/of.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/mips/ralink/of.c
+++ b/arch/mips/ralink/of.c
@@ -74,7 +74,7 @@ void __init device_tree_init(void)
 	unflatten_device_tree();
 
 	/* free the space reserved for the dt blob */
-	free_bootmem(base, size);
+	//free_bootmem(base, size);
 }
 
 void __init plat_mem_setup(void)
